src/pages/resultsView/enrichments/ExpressionEnrichmentsContainer.tsx

```diff
@@ -177,7 +177,7 @@ export default class ExpressionEnrichmentContainer extends React.Component<IExpr
 
     public render() {
 
-        if (this.props.data.length === 0) {
+        if (!this.props.data || this.props.data!.length === 0) {
```

Comment:
You could use `_.isEmpty(this.props.data!)`, which checks for undefined and length === 0

Reply:
good point, changed, thanks!

Reply:
@khzhu looks like its not updated

Reply:
Have not checked in, @kalletlak,